cmd/inspector/main.go

```diff
@@ -121,6 +122,13 @@ var (
 		Args:  cobra.MinimumNArgs(1),
 		RunE:  CreateCSVStats,
 	}
+	objectStatsCmd = &cobra.Command{
+		// TODO: add args to usage
+		Use:   "objectstats <path>",
```

Comment:
`object-health`, also the naming above is very in consistent.
`<encrypted-path>` to make clear that it must be already encrypted

Reply:
done